src/register-org/components/company-house-details/company-house-details.component.html

```diff
@@ -28,10 +28,10 @@ <h1 class="govuk-heading-xl">What is your company name and Companies House numbe
             Enter the 8-digit Companies House Number (optional)
           </label>
           <div id="company-house-number-hint" class="govuk-hint">
-            This helps HMCTS to verify your company on our System
+            This helps HMCTS to verify your organisation on our System
```

Comment:
@johnbenjamin-hmcts - not related to your changes, but I'm not sure why we're capitalising the 's' in "System"... I've seen this elsewhere and would suggest it's just "system"

Reply:
@Daniel-Lam - I have updated the code now. Could you please re-review?

Reply:
@johnbenjamin-hmcts - all good